test/image/mocks/gl3d_surface_contour_precision.json

```diff
@@ -0,0 +1,33 @@
+{
+    "data": [
```

Comment:
I love this mock! Unfortunately I'm seeing some more weird behavior when I play with it... all these axis features flicker above & below the data.
<img width="415" alt="screen shot 2018-12-06 at 6 29 11 pm" src="https://user-images.githubusercontent.com/2678795/49618112-46e24280-f985-11e8-9274-715cee0f44cd.png">

Reply:
Those two mocks are intended to possibly show improvements and at the same time certain remaining problems related to data precision. Are you experiencing this on the latest release as well? I've just fetch master as there was another [PR](#3302) that could help.

Reply:
Concerning @alexcjohnson observation, I tested another new mock with CI on current master branch (i.e. without these changes applied); and the same z-fighting existed [there](https://20938-45646037-gh.circle-artifacts.com/0/test_images/gl3d_parametric_surface_data_precision.png). Now I may confirm that this problem is not related to the changes of this branch related to objectOffset.

Reply:
OK. That issue is now reported in #3346.